src/main/java/InterfaceAdapters/ScheduleManager.java

```diff
@@ -18,56 +19,63 @@
 
 public class ScheduleManager {
 
-    // attempts to fix these style warnings have resulted in problems in other classes
-    private HashMap<Person, Schedule> schedules;
-    private final ScheduleEditor editor;
-    private final ScheduleComparer comparer;
+//    // attempts to fix these style warnings have resulted in problems in other classes
+//    private HashMap<Person, Schedule> schedules;
+    private ScheduleEditor editor;
+    private ScheduleComparer comparer;
+    private UserList data;
+    private FriendAdder adder;
 
     public ScheduleManager() {
         editor = new ScheduleEditor();
         comparer = new ScheduleComparer();
-        schedules = new HashMap<>();
+//        schedules = new HashMap<>();
     }
 
-    /**
-     * Adds an instance of Person and Schedule to the schedules hashmap
-     * @param user1 the main user's Person
-     * @param user2 the friend's Person
-     */
+//    /**
+//     * Compares two people's schedules and returns a schedule showing what times both
+//     * are busy.
+//     * @param user1 the main user's Person
+//     * @param user2 the friend's Person
+//     */
+    // currently uncertain how we're going to be using this class
+//    public Schedule compare(String user1, String user2) {// creating hard dependencies -- i don't think we're supposed to do this?
+//        return comparer.compare(user1, user2);
+//    }
 
-    public Schedule compare(Person user1, Person user2) {// creating hard dependencies -- i don't think we're supposed to do this?
-        return comparer.compare(user1, user2);
-    }
-
-    public void addEvent(String eventType, String eventName, String eventDay, int eventStartTime, int eventEndTime, Person user){
-        editor.addEvent(eventType, eventName, eventDay, eventStartTime, eventEndTime, user);
+    public void addEvent(String eventType, String eventName, String eventDay, int eventStartTime,
+                         int eventEndTime, String user){
+        editor.addEvent(eventType, eventName, eventDay, eventStartTime, eventEndTime, data.getUser(user));
     }
 
     // should we change this to give it an event
-    public void removeEvent(String eventName, String eventDay, Person user) {
-        editor.removeEvent(eventName, eventDay, user);
+    public void removeEvent(String eventName, String eventDay, String user) {
+        editor.removeEvent(eventName, eventDay, data.getUser(user));
     }
 
-    public void sendFR(Person user1, String username2){
+    // todo: edit this to return a boolean if a friend is successfully added??
```

Comment:
Maybe the boolean should be in the acceptFR method to show when user2 accepted user1's friend request?

Reply:
ohh yeah that could probably do with a boolean. although i think it would be good for this to return a boolean as well just so we always know whether a "send friend request" worked or not -- you might click "send request" and then think you sent a request but you actually didn't because that user doesn't exist or something like that

Reply:
ah yeah true so they could both return a boolean, each for different reasons. sendFR for notifying user1 that their friend request was sent and acceptFR to notify user1 that user2 accepted the request. right?

Reply:
yes! if you think that makes sense, i'll add it in